lib/meadow/elasticsearch_diff_store.ex

```diff
@@ -30,7 +30,7 @@ defmodule Meadow.ElasticsearchDiffStore do
     |> Repo.stream()
     |> Stream.chunk_every(@chunk_size)
     |> Stream.flat_map(fn chunk ->
-      Repo.preload(chunk, :collection)
+      Repo.preload(chunk, [:collection, :file_sets])
```

Comment:
Is this efficient? I'm only asking because I started to do this yesterday and then started googling and it seemed like it was suggested to manually join over doing this. But I trust you if you've looked into it.

Reply:
When I was reading up on this I saw a Twitter thread by Michał Muskała that hopefully clarifies it a bit: https://twitter.com/michalmuskala/status/1118119934758928385.